src/main/java/com/exactpro/epfast/template/implementations/TemplatesClass.java

```diff
@@ -0,0 +1,16 @@
+package com.exactpro.epfast.template.implementations;
+
+import com.exactpro.epfast.template.Template;
+import com.exactpro.epfast.template.Templates;
+
+import java.util.ArrayList;
+
+public class TemplatesClass implements Templates {
+
+    private final ArrayList<Template> templates = new ArrayList<>();
+
+    @Override
+    public ArrayList<Template> getTemplates() {
```

Comment:
`List<TemplateClass>` (it will be renamed back to `Template`, but it should be that is currently `TemplateClass`).

Change return type in interface to `List<? extends Template>`. Do the same for other methods which return list. I missed that then we created interfaces.